config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'app#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :app
```

Comment:
- [ ] Kindly make sure that you have the following routes:
```  
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
```

Reply:
* This issue hasn't been resolved, You need to create routes for users and posts 
* you can use write your routes like this 👇 
 ```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
```
![image](https://user-images.githubusercontent.com/52497246/204529350-e388e1f9-a8e3-45ac-92c2-a6f37f148570.png)